akka-persistence/src/main/resources/reference.conf

```diff
@@ -71,6 +72,7 @@ akka.persistence {
                 parallelism-max = 8
             }
         }
+        # Default dispatcher for streaming snapshot IO
         default-stream-dispatcher {
```

Comment:
hm ok, it is in config `stream-dispatcher = "akka.persistence.dispatchers.default-stream-dispatcher"`; nevermind, keep as is